packages/react-components/src/components/Picker/Trigger.tsx

```diff
@@ -14,7 +14,7 @@ const baseClass = 'picker-trigger';
 export type TriggerSize = 'compact' | 'medium' | 'large';
 
 export interface ITriggerProps {
-  disabledSearch: boolean;
+  isSearchDisabled: boolean;
```

Comment:
Here we have `isSearchDisabled` as this is an internal part of `Picker` component, where our naming convention perfectly fits, the consumer doesn't see it.

Reply:
Fine for now, we'll rethink this later